src/codegen.cpp

```diff
@@ -4865,6 +4856,21 @@ static GlobalVariable *global_to_llvm(const std::string &cname, void *addr, Modu
     add_named_global(gv, addr);
     return gv;
 }
+std::map<jl_value_t**, GlobalVariable*> gv_for_global;
+static GlobalVariable *global_to_llvm(const std::string &cname, jl_value_t **addr, Module *m)
```

Comment:
Given that we have global variables of type other than `jl_value_t*` (`jl_datatype_t*` for example) I think we should be more explicit here.